bugbot/rules/file_crash_bug.py

```diff
@@ -198,10 +234,7 @@ def get_bugs(self, date):
                 if bug_data["regressed_by"] and not updates:
                     # If we don't set the nightly flag here, the bot will set it
                     # later as part of `regression_new_set_nightly_affected` rule.
-                    nightly_flag = utils.get_flag(
-                        self.nightly_version, "status", "nightly"
-                    )
-                    bug_data[nightly_flag] = "affected"
+                    bug_data[self.nightly_status_flag] = "affected"
```

Comment:
I don't remember, is there a reason why we only set Nightly as "affected" when we have a "regressed_by"?

Reply:
In case we did not set any status flag based on the "regressed by" (e.g., because no status flags were set on the regressor bug), this is just replicating what `regression_new_set_nightly_affected` will do to avoid the noise of having the bot doing it later separately:

https://github.com/mozilla/bugbot/blob/bf8cad1b18c47d08fa9a00fad1cdb84a3e7f5749/bugbot/rules/file_crash_bug.py#L235-L236

Reply:
Couldn't we set Nightly as "affected" always, even if "regressed by" is not present?

Reply:
We could set "nightly" as affected without adding the "regression" keyword, as we are not sure it is a regression but we are sure nightly is affected